stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -395,13 +401,24 @@ class CardNumberEditText internal constructor(
                 if (isComplete(wasCardNumberValid)) {
                     completionCallback()
                 }
+            }
+            // Partial card number entered and brand is not yet determine, but possible.
+            else if (isPossibleCardBrand(unvalidatedCardNumber.normalized)) {
```

Comment:
Not sure why ktlint isn't catching this. Should be `} else if (...) {`

Also why would this condition cause an error to be shown?

Reply:
If you enter a 1 or a 0, that is not a possible card brand

Reply:
Should it be `!isPossibleCardBrand()` then?

Reply:
The name here is actually wrong, should be isImpossibleCardBrand